be/src/exec/pipeline/exchange/exchange_merge_sort_source_operator.h

```diff
@@ -0,0 +1,95 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+#pragma once
+
+#include <atomic>
+
+#include "exec/pipeline/source_operator.h"
+
+namespace starrocks {
+class DataStreamRecvr;
+class RowDescriptor;
+class SortExecExprs;
+namespace pipeline {
+class ExchangeMergeSortSourceOperator : public SourceOperator {
+public:
+    ExchangeMergeSortSourceOperator(int32_t id, int32_t plan_node_id, int32_t num_sender, const RowDescriptor& row_desc,
```

Comment:
Needn't pass RowDescriptor, you could use empty RowDescriptor, Currently，in expression compute, we don't use RowDescriptor any more

Reply:
How to use a empty RowDescriptor? I tried, but it was wrong.
`Check failed: false unreachable path`